Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.3.1
```

Comment:
Inherit from `ruby:2.3.1-alpine` instead of the plain ruby image. (please see: [https://hub.docker.com/_/alpine/](https://hub.docker.com/_/alpine/).)